### ISSUE TITLE
Does CaffeNet show top-5 accuracy numbers for the validation data?

### ISSUE THREAD

**Issue by cuihenggang** [BVLC/caffe#2673]
Hi,

I'm trainning the CaffeNet on the Ilsvrc12 dataset. During the training, I got messages like:

...
Iteration 1000, Testing net (#0)
Test net output #0: accuracy = 0.0286202
Test net output #1: loss = 10.2503 (* 1 = 10.2503 loss)
...

I'm wondering is that possible to show both top-5 accuracy and top-1 accuracy?

Thanks,
Cui

**Comment by longjon** [comment]
Closing as this looks like a usage issue/request for help.

This tracker is reserved for specific Caffe development issues and bugs; please ask usage questions on the [caffe-users](https://groups.google.com/forum/#!forum/caffe-users) list.

For more information, see our [contributing guide](https://github.com/BVLC/caffe/blob/master/CONTRIBUTING.md).

Thanks!

**Comment by GeorgeThantZin** [comment]
go here : https://stackoverflow.com/questions/33098987/calculating-top-5-error-rate-in-caffe
